### ISSUE TITLE
XKCD theme for XChart

### ISSUE THREAD

**Issue by ouryperd** [knowm/XChart#434]
Feature request for adding an xkcd (www.xkcd.com) style / theme to charts.
The rationale: https://www.chrisstucchio.com/blog/2014/why_xkcd_style_graphs_are_important.html
matplotlib has done it very succesfully: https://matplotlib.org/xkcd/examples/showcase/xkcd.html

**Comment by Mr14huashao** [comment]
@timmolter 
This theme is completely different from the previous three themes of XChart.

 README.md: https://github.com/knowm/XChart#chart-themes

**Comment by timmolter** [comment]
Yes, because it's not a matter of styling a straight line; the lines are actually wavy. We'd not just be changing stroke properties, but have to  custom draw the lines probably. However I think it would be worth doing and I've been wanting to do this for years now. Let me look into it a bit...

**Comment by timmolter** [comment]
https://gist.github.com/FlorianCassayre/cd1c9fc5a7e6194b18dccb800de53341
https://gist.github.com/bmorris591/d631b56bf4ee6ce8fe3b42227e8dcea0
